qiskit/circuit/library/n_local/n_local.py

```diff
@@ -1009,7 +1020,10 @@ def get_entangler_map(
     if entanglement == "pairwise" and num_block_qubits != 2:
         raise ValueError("Pairwise entanglement is only defined for blocks of 2 qubits.")
 
-    if entanglement == "full":
+    if entanglement == "full" or entanglement == "full_explicit":
```

Comment:
This would be more concise, if you want to use it 🙂 
```suggestion
    if entanglement in ["full", "full_explicit"]:
```